requirements.txt

```diff
@@ -1,4 +1,3 @@
-certifi==2016.2.28
-django
 gunicorn
-django-heroku
+flask
+requests
```

Comment:
Your app uses the `flask` and `requests` libraries so if you include them in requirements.txt they'll be installed on heroku.

I removed the other stuff cuz it worked without them, but idk if you were planning to use them?